cluster/aws/util.sh

```diff
@@ -352,6 +352,34 @@ function wait-for-instance-running {
   done
 }
 
+function allocate-elastic-ip {
+  $AWS_CMD allocate-address --domain vpc --output text | cut -f3
```

Comment:
I'm not sure what you meant. ```"${AWS_CMD}"``` will be treated as string (so not executed). ```"$(AWS_CMD)"``` literally as command.

```bash
#!/usr/bin/env bash
AWS_CMD="ls -la"
echo "${AWS_CMD}" # => ls -la
echo "$(AWS_CMD)" # => ./cli.sh: line 4: AWS_CMD: command not found
```